src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrentStalledPolicy.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+
+namespace NzbDrone.Core.Download.Clients.QBittorrent;
+
+public interface IQBittorrentStalledPolicy
+{
+    void Apply(QBittorrentTorrent torrent, DownloadClientItem ditem, QBittorrentSettings settings);
+}
+
+public interface IDateTimeProvider
```

Comment:
This is really only needed for testing so that we can verify that time is accurate to the second which isn't necessary for the tests and not something we do in other places where we're comparing times. This code should be converted to `DateTime.UtcNow` and the tests updated to fall within the bounds of the thresholds even if they take part of a second to complete.